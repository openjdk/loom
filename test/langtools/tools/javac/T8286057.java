/*
 * @test /nodynamiccopyright/
 * @bug 8286057
 * @summary Make javac error on a generic enum friendlier
 *
 * @compile/fail/ref=T8286057.out -XDrawDiagnostics T8286057.java
 */

class EnumsCantBeGeneric {
  public enum E1<> {}
  public enum E2<T> {}
  public enum E3<T, T> {}
}
