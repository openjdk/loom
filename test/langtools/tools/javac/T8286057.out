T8286057.java:10:17: compiler.err.enum.cant.be.generic
T8286057.java:11:18: compiler.err.enum.cant.be.generic
T8286057.java:12:18: compiler.err.enum.cant.be.generic
3 errors